Realm/Realm.Unity/Editor/Common/ImportedReferences.cs

```diff
@@ -143,7 +143,7 @@ internal abstract class ImportedReferences
 
         public TypeReference RealmSchema_PropertyType { get; private set; }
 
-        public MethodReference UnityEngine_RuntimeInitializeOnLoadAttribute_Constructor { get; private set; }
+        public MethodReference UnityEngine_PreserveAttribute_Constructor { get; private set; }
```

Comment:
You don't need the unity preserve attribute. You can use [PreserveAttribute_Constructor](https://github.com/realm/realm-dotnet/blob/aff5bb2bd02e029cc847c0d16f9bfef47f713802/Realm/Realm.Unity/Editor/Common/ImportedReferences.cs#L120).

Reply:
Oh, I missed that. I'll update.